core/vm/gas_table.go

```diff
@@ -519,6 +593,10 @@ func gasSelfdestruct(evm *EVM, contract *Contract, stack *Stack, mem *Memory, me
 		}
 	}
 
+	if evm.Accesses != nil {
+		log.Warn("verkle witnesses accumulation not supported for selfdestruct")
```

Comment:
please add a note to say that this should be eventually turned into a panic because selfdestruct isn't supposed to exist by the time verkle trees are rolled out.